sdk/ai/azure-ai-agents/CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 # Release History
 
```

Comment:
Let's add update regarding create run where following parameters has been added.

keyword tool_resources: The overridden enabled tool resources that the agent should use to run
         the thread. Default value is None.